client/src/components/Modal/Modal.jsx

```diff
@@ -0,0 +1,73 @@
+import React, { useContext } from 'react';
+import {
+  Container,
+  Overlay,
+  CloseOut,
+  TopBar,
+  Header,
+  TitleText,
+  ModalContentBorder,
+  ModalContentWrapper,
+  LeftSideContent,
+  TagTitle,
+  TagWrapper,
+  SideSeparator,
+  ToolTip,
+  QuestionMarkHover,
+  TagList,
+  TagsSpacer,
+  SpanTags,
+  RightSideContent,
+  SignInButton,
+  SignInHeader,
+  SignInInfo,
+} from './styles';
+import GamesContext from '../../context';
+
+function Modal() {
+  const { showModal, setModal, sidePanelInfo } = useContext(GamesContext);
+  return (
+    <>
+      <Overlay showModal={showModal} onClick={() => setModal(false)} />
+      <Container showModal={showModal}>
+        <TopBar />
+        <Header>
+          <CloseOut onClick={() => setModal(false)}>X</CloseOut>
+          <TitleText>View and edit tags for this product</TitleText>
+        </Header>
+        <ModalContentBorder>
+          <ModalContentWrapper>
+            <TagWrapper>
+              <SideSeparator />
+              <LeftSideContent>
+                <TagTitle>
+                  Popular user-defined tags for this product:
+                  <QuestionMarkHover>
+                    (?)
+                    <ToolTip>
+                      These are tags applied to the product by the most users.
+                      You can click a tag to find other products with that tag applied.
+                    </ToolTip>
+                  </QuestionMarkHover>
+                </TagTitle>
+                {sidePanelInfo.popular_tags.map((tag, i) => (
+                  <TagList key={`${i}a`}>
```

Comment:
Creative re-use of key with "i" w/ a map, may use this sometime if I'm trying to render more than 1 component from a map